manifests/overlays/prod/configs/argo_rbac_cm/policy.csv

```diff
@@ -61,4 +61,12 @@ p, role:aiops-admin, applications, sync, aiops/*, allow
 p, role:aiops-admin, applications, override, aiops/*, allow
 p, role:aiops-admin, applications, action/*, aiops/*, allow
 g, aicoe-aiops-devops, role:argocd-admin
-g, aicoe-aiops-devops, role:aiops-admin
\ No newline at end of file
+g, aicoe-aiops-devops, role:aiops-admin
+
+p, role:ccx-admin, applications, get, ccx/*, allow
+p, role:ccx-admin, applications, update, ccx/*, allow
+p, role:ccx-admin, applications, delete, ccx/*, allow
+p, role:ccx-admin, applications, sync, ccx/*, allow
+p, role:ccx-admin, applications, override, ccx/*, allow
+p, role:ccx-admin, applications, action/*, ccx/*, allow
+g, ccx-dev, role:ccx-admin
```

Comment:
the configs are a special situation where we don't add the newline, do you mind removing it again

Reply:
actually I think it's fine for policy.csv

Reply:
Good to know, I've fixed it anyways. :+1: